spec/models/partner_request_spec.rb

```diff
@@ -12,6 +12,29 @@
     end
   end
 
+  describe "a partner with a form 990" do
+    it "returns proof of form 990" do
+      partner_stubbed = build(:partner, :with_990_attached)
+      expect(build_stubbed(:partner_request, partner: partner_stubbed).partner.export_json.dig(:stability, :form_990_link)).to include("f990.pdf")
+    end
+  end
+
+  describe "a partner must contain" do
+    it "proof_of_agency_status" do
+      partner_stubbed = build(:partner, :with_status_proof)
+      expect(build_stubbed(:partner_request, partner: partner_stubbed).partner.export_json[:proof_of_agency_status]).to include("status_proof.pdf")
+    end
+  end
+
+  describe "a partner must contain" do
```

Comment:
`describe` blocks can contain multiple `it` blocks, so you don't need a second one here.